server/sonar-server/src/main/java/org/sonar/server/authentication/OAuthCsrfVerifier.java

```diff
@@ -44,8 +46,12 @@ public String generateState(HttpServletRequest request, HttpServletResponse resp
     return state;
   }
 
-  public void verifyState(HttpServletRequest request, HttpServletResponse response) {
-    Cookie cookie = findCookie(CSRF_STATE_COOKIE, request).orElseThrow(() -> new UnauthorizedException(format("Cookie '%s' is missing", CSRF_STATE_COOKIE)));
+  public void verifyState(HttpServletRequest request, HttpServletResponse response, String providerName) {
+    Cookie cookie = findCookie(CSRF_STATE_COOKIE, request)
+      .orElseThrow(() -> AuthenticationException.newBuilder()
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this lambda with a method reference. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1612)